src/fe-text/irssi.c

```diff
@@ -223,6 +223,7 @@ static void textui_deinit(void)
 #endif
 
         dirty_check(); /* one last time to print any quit messages */
```

Comment:
looks like a space here

Reply:
I didn't touch the line

Reply:
But you were on the adjacent line, uparrow once and delete the spaces :P

Reply:
We're going to run `clang-format` over the whole codebase anyway, don't worry about the stylistic details :)